election-api-scala/test/controllers/ResultsControllerSpec.scala

```diff
@@ -21,47 +19,37 @@ class ResultsControllerSpec extends PlaySpec with GuiceOneAppPerTest with Inject
 
     "Test first 5 results" in {
       val scoreboard = runXResults(5)
-      scoreboard match {
-        case Some(sc) => matchScoreboard(scoreboard = sc, LabSeats = 4, LDSeats = 1)
-        case None => fail("Didn't return a scoreboard")
-      }
+      scoreboard must not be empty
+      // LAB = 4
+      // LD = 1
+      // winner = noone
     }
 
     "First 100 results" in {
       val scoreboard = runXResults(100)
-      scoreboard match {
-        case Some(sc) => matchScoreboard(scoreboard = sc, LabSeats = 56, LDSeats = 12, ConSeats = 31)
-        case None => fail("Didn't return a scoreboard")
-      }
+      scoreboard must not be empty
+      // LD == 12
+      // LAB == 56
+      // CON == 31
+      // winner = noone
     }
 
     "First 554 results" in {
       val scoreboard = runXResults(554)
-      scoreboard match {
-        case Some(sc) => matchScoreboard(scoreboard = sc, LabSeats = 325, LDSeats = 52, ConSeats = 167, winner = Some("LAB"))
-        case None => fail("Didn't return a scoreboard")
-      }
+      scoreboard must not be empty
+      // LD == 52
+      // LAB = 325
+      // CON = 167
+      // winner = LAB
     }
 
     "All results" in {
       val scoreboard = runXResults(650)
-      scoreboard match {
-        case Some(sc) => matchScoreboard(scoreboard = sc, LabSeats = 349, LDSeats = 62, ConSeats = 210, winner = Some("LAB"))
-        case None => fail("Didn't return a scoreboard")
-      }
-    }
-
-    def matchScoreboard(scoreboard: Scoreboard, LabSeats: Int = 0,
-                        LDSeats: Int = 0, ConSeats: Int = 0, winner: Option[String] = None) {
-      var ld, lab, con = 0
-
-      //TODO: set the seats by party
-
-      ld mustEqual LDSeats
-      lab mustEqual LabSeats
-      con mustEqual ConSeats
-//      something mustEqual winner
-      fail("Implement me")
+      scoreboard must not be empty
+      // LD == 62
+      // LAB == 349
+      // CON == 210
+      // winner = LAB
     }
```

Comment:
👍 on having comments stating assertions needed & letting the candidate implement (as in other languages)